ext/date/date_core.c

```diff
@@ -9294,6 +8993,54 @@ Init_date_core(void)
      *     d += 1			#=> #<Date: 2001-02-04 ...>
      *     d.strftime('%a %d %b %Y')	#=> "Sun 04 Feb 2001"
      *
+     * === Argument +start+
+     *
+     * Certain calculations and comparisons for a \Date object
+     * are affected by what the object considers to have been
+     * the changeover date from the
+     * {Julian}[https://en.wikipedia.org/wiki/Julian_calendar] to the
+     * {Gregorian}[https://en.wikipedia.org/wiki/Gregorian_calendar]
+     * calendar;
+     * this is set by argument +start+ when the object is created:
+     *
+     * - Dates before the changeover are considered to be Julian.
+     * - Dates after the changeover are considered to be Gregorian:
```

Comment:
I think this is a typo

```suggestion
     * - Dates after the changeover are considered to be Gregorian.
```

Reply:
Fixed.